components/SwiperSection.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div :class="`swiper-section-${uuid}`" style="width: 100%">
+    <skeleton-home-section v-if="loading" :card-shape="shape" />
+    <v-col v-show="items && items.length > 0" class="swiper-section">
+      <div class="d-flex">
+        <h1
+          class="text-h5 font-weight-light header mt-1 pl-3 pb-2"
+          :class="{ 'header-white-mode': !$vuetify.theme.dark }"
+        >
+          <span class="pl-3">{{ title }}</span>
+        </h1>
+        <v-spacer />
+        <v-btn class="swiper-prev" icon>
+          <v-icon>mdi-arrow-left</v-icon>
+        </v-btn>
+        <v-btn class="swiper-next mr-2" icon>
+          <v-icon>mdi-arrow-right</v-icon>
+        </v-btn>
+      </div>
+
+      <swiper class="swiper" :options="swiperOptions">
+        <swiper-slide v-for="item in items" :key="item.Id">
+          <card :shape="shape" :item="item" />
+        </swiper-slide>
+      </swiper>
+    </v-col>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { SwiperOptions } from 'swiper';
+import { v4 as uuidv4 } from 'uuid';
+import { BaseItemDto } from '@jellyfin/client-axios';
+
+export default Vue.extend({
+  props: {
+    loading: {
+      type: Boolean,
+      required: true
+    },
+    title: {
+      type: String,
+      required: true
+    },
+    items: {
+      type: Object as () => BaseItemDto[],
```

Comment:
This is buggy. When rendering home sections, this can be be undefined, since they do not yet exist in the vuex store (for latest media). 

We could do something like:
`
      type: (undefined || Array) as () => BaseItemDto[],
`
But I couldn't get this to work properly.

Alternatively, specify it as not required, and have a default of an empty array.